manifests/init.pp

```diff
@@ -635,7 +635,9 @@
     path => '/bin:/sbin:/usr/bin:/usr/sbin',
   }
 
+  $confd_command = ['mkdir', $confd_dir]
   exec { "mkdir ${confd_dir}":
+    command => $confd_command,
```

Comment:
Do you need to define an additional variable for this? I'd think this is the same
```suggestion
  exec { "mkdir ${confd_dir}":
    command =>  ['mkdir', $confd_dir],
```

Reply:
It's mainly a stylistic choice to keep everything consistent.
While this instance and a few others would read fine included like you suggested there are others that read significantly better separated out and so I have them all separated out for consistency.